File: Loop.py
#rows=5
#for i in range(1, rows+1):
    #print(" "*(rows-i) + "*" * (2*i-1))

#di py range akhir itu tidak dihitung. jadi misal rows=5, bakal terhitungnya 4. 
# makanya di sana ditulis rows+1,biar tetap 5

#looping 1: print spasi sebanyak rows-i (5-1) yaitu 4. lalu baru print *

rows= 5
while rows >= 1:
    print("*" * rows)
    rows -= 1

# rows -= 1 artinya rows = rows -1

File: Loop.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    int rows=10;
    for(int i=1;i<=rows;i++) {
        for(int j=1; j<=rows-i; j++){
            cout <<" ";
        }
        for(int k=1; k<=2*i-1; k++){
           cout <<"*";
      }
       cout << endl;
    }
    return 0;
}

// int main() {
// int rows=5;
// while(rows >= 1){
//   cout << string(rows-1, '*') << endl;
//    rows--;
// }
// return 0;
// }


// int main(){
//     int rows=5;
//     int i=1;
//     do {
//         int j=1;
//         do {
//             cout <<" ";
//             j++;
//         } while(j<=rows-i);
//         int k=1;
//         do {
//             cout <<"*";
//             k++;
//         } while(k<=i);
//         cout << endl;
//         i++;
//     } while(i<=rows);
//     return 0;
// }